packages/atomic/src/components/search/facets/atomic-numeric-facet/atomic-numeric-facet.tsx

```diff
@@ -87,7 +70,7 @@ interface NumericRangeWithLabel extends NumericRangeRequest {
  */
 @Component({
   tag: 'atomic-numeric-facet',
-  styleUrl: 'atomic-numeric-facet.pcss',
+  styleUrl: '../../../common/facets/numeric-facet-common.pcss',
```

Comment:
I personally rather keep a pcss file in the same folder & import the common styles from the pcss. This way if there is ever some different styles from 1 facet to the other, we won't need to change this property. The pcss file in the same folder is also the place I think most collaborators would look to first, before checking this property.

What do you think?

Reply:
I agree

Reply:
no worries! will update